src/modules/RA/DataLayer/components/tabs/PopupTab/index.js

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useCallback } from 'react';
+import { useField, useForm } from 'react-final-form';
+import {
+  TextInput,
+  // NumberInput,
```

Comment:
Do you want to keep it ?

Reply:
No, it is an oversight of my part. I will fix it

Reply:
Done.